src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -218,7 +218,7 @@ void RaftReplDev::on_fetch_data_received(intrusive< sisl::GenericRpcData >& rpc_
             // accumulate the sgs for later use (send back to the requester));
             sgs_vec.push_back(sgs);
 
-            async_read(local_blkid, sgs, total_size).thenValue([this, &ctx](auto&& err) {
+            async_read(local_blkid, sgs, total_size).thenValue([this, ctx](auto&& err) {
```

Comment:
this is not needed as it is sync wait in the same function and lambda is destroyed. It will soon be replaced with fut.collectAll in comming PR.